tests/rule_based_profiler/test_toolkit.py

```diff
@@ -0,0 +1,28 @@
+from unittest import mock
+
+from great_expectations.data_context.types.resource_identifiers import (
+    ConfigurationIdentifier,
+)
+from great_expectations.rule_based_profiler.toolkit import list_profilers
+
+
+@mock.patch("great_expectations.data_context.store.ProfilerStore")
+def test_list_profilers(mock_profiler_store: mock.MagicMock):
+    store = mock_profiler_store()
+    keys = ["a", "b", "c"]
+    store.list_keys.return_value = [ConfigurationIdentifier(char) for char in keys]
+    res = list_profilers(store, ge_cloud_mode=False)
+
+    assert res == keys
+    assert store.list_keys.called
+
+
+@mock.patch("great_expectations.data_context.store.ProfilerStore")
+def test_list_profilers_in_cloud_mode(mock_profiler_store: mock.MagicMock):
```

Comment:
Just curious, what is the difference between the `cloud_mode` test and the non-`cloud_mode` test?

Reply:
Just testing both paths to ensure 100% coverage of the method.

When in cloud mode, `list_keys` returns a list of strings. In non-cloud mode, this is a list of `ConfigurationIdentifier` (which requires the extra step to extract the string value).

This is an implementation detail at the store level and mirrors the Checkpoint toolkit.